FaceCluster[] faces
ClusterInCamera[] inCamera
time stamp